src/ngx_http_set_hashed_upstream.c

```diff
@@ -1,27 +1,71 @@
 #ifndef DDEBUG
 #define DDEBUG 0
 #endif
+
 #include "ddebug.h"
 
 #include "ngx_http_set_hashed_upstream.h"
 
+#define HASH_VNODES           160
 
 ngx_uint_t
-ngx_http_set_misc_apply_distribution(ngx_log_t *log, ngx_uint_t hash,
-    ndk_upstream_list_t *ul, ngx_http_set_misc_distribution_t type)
-{
+ngx_http_set_misc_apply_distribution(
+        ngx_log_t *log,
+        ngx_uint_t hash,
+        ngx_http_set_hashed_upstream_conf *hash_conf,
+        ngx_http_set_misc_distribution_t type) {
+
```

Comment:
two style thing: 1. we usually try to use lesser lines, `log`, `hash` can follow the function name in the same line.
2. `{` should in a new line.